dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs

```diff
@@ -47,12 +46,14 @@ public void SampleFunctionalTest1()
                     Assert.AreEqual("Home Page - My ASP.NET Application", driver.Title, "Expected title to be 'Home Page - My ASP.NET Application'");
                     break;
                 }
-                catch
+                catch(Exception e)
                 {
                     var currentTimestamp = DateTime.Now.Millisecond;
                     if(currentTimestamp > endTimestamp)
                     {
-                        throw;
+                        Console.Write("##vso[task.logissue type=error;]Test SampleFunctionalTest1 failed with error: " + e.ToString());
+                        throw e;
+                        break;
```

Comment:
why break is needed?

Reply:
It doesn't seem to break out of the loop without break statement.. the test will continue running indefinitely without break

Reply:
I've updated